projects/shared/jest.config.js

```diff
@@ -1,7 +1,5 @@
-require('jest-preset-angular/ngcc-jest-processor');
-
 module.exports = {
   preset: 'jest-preset-angular',
   setupFilesAfterEnv: ['<rootDir>/projects/shared/src/setup-jest.ts'],
-  testPathIgnorePatterns: ['<rootDir>/projects/shared/cypress/']
+  testPathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/projects/shared/cypress/'],
```

Comment:
From the error message of the test, 
I thought that it was necessary to ignore other than cosmosclient, so I fixed it like this in local.

```
  testPathIgnorePatterns: [
    '<rootDir>/node_modules/(?!(cosmosclient)/)',
    '<rootDir>/projects/shared/cypress/',
  ],
```
[https://jestjs.io/docs/tutorial-react-native#transformignorepatterns-customization](https://jestjs.io/docs/tutorial-react-native#transformignorepatterns-customization)

Reply:
However, the test still fails. 
Is this fixed `testPathIgnorePatterns` wrong?